src/components/m-display-value.vue

```diff
@@ -0,0 +1,87 @@
+
+<template>
+  <p :class="numberColorClass">{{ valueDisplayed }}</p>
+</template>
+
+<script>
+import { roundNumber } from "@/utils/number";
+
+type SignType = "euro" | "percent" | "point" | "null"
+
+interface NumberOptions {
+  maximumFractionDigits: number,
+  currency?: string,
+  style?: string,
+}
+
+export default {
+  name: 'MDisplayValue',
+  props: {
+    value: {
+      type: Number,
+      default: 0
+    },
+    digitsAfterDecimal: {
+      type: Number,
+      default: 0
+    },
+    signType: {
+      type: String as () => SignType,
+      default: ''
+    },
+    showPlusSign: {
+      type: Boolean,
+      default: false
+    },
+    showColors: {
+      type: Boolean,
+      default: false
+    }
+  },
+  computed: {
+    valueDisplayed(): string {
+    const options: NumberOptions = {
+      maximumFractionDigits: this.digitsAfterDecimal
+    }
+    let type = 1
+    if(this.signType === 'percent') type = 100
+    const number = roundNumber(this.value * type, 2)
```

Comment:
Le type ça m'a paru bizarre que ce soit un nombre. Est-ce qu'un truc comme ça ne serait pas plus compréhensible a premier abord ?

```suggestion
    let type = 1
    if(this.signType === 'percent') type = 100
    const number = roundNumber(this.signType === 'percent' ? this.value * 100 : this.value, 2)
```